src/agentscope/rag/search_knowledge.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+"""
+This module is an integration of the Llama index RAG
+into AgentScope package
+"""
+import json
+import os.path
+from typing import Any, Optional, List
+
+from llama_index.core.data_structs import Node
+
+# search
+from llama_index.core.schema import NodeWithScore
+from loguru import logger
+
+from agentscope.constants import (
+    DEFAULT_CHUNK_SIZE,
+)
+from agentscope.service import load_web
+from agentscope.utils.token_utils import count_openai_token
```

Comment:
Use `agentscope.tokens` instead, the `agentscope.utils.token_utils` will be removed soon